lib/features/base/base_controller.dart

```diff
@@ -372,6 +375,27 @@ abstract class BaseController extends GetxController
     }
   }
 
+  void injectWebSocket(Session? session, AccountId? accountId) {
+    try {
+      requireCapability(
+        session!,
+        accountId!,
+        [
+          CapabilityIdentifier.jmapWebSocket,
+          CapabilityIdentifier.jmapWebSocketTicket
+        ]
+      );
+      final wsCapability = session.getCapabilityProperties<WebSocketCapability>(
+        accountId,
+        CapabilityIdentifier.jmapWebSocket);
+      if (wsCapability?.supportsPush != true) return;
```

Comment:
throw exception to indicate: push is not supported

Reply:
Done